src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -45,6 +45,9 @@ public boolean update(int expiryTime) {
     public boolean hireRetriever(Retriever retriever) {
         if (data.getAvailableSlots() <= 0) return false;
         //retriever.getCost(); // TODO: check cost
+        if(retriever.getCreditCost() > main.statsManager.credits || retriever.getUridiumCost() > main.statsManager.uridium || retriever.getPermitCost() > data.getPermit()){
```

Comment:
split line, it's too long
```suggestion
        if (retriever.getCreditCost() > main.statsManager.credits || 
                retriever.getUridiumCost() > main.statsManager.uridium || 
                retriever.getPermitCost() > data.getPermit()) {
```